WordPress/WordPressUITests/WordPressUITests.xctestplan

```diff
@@ -22,18 +22,13 @@
     {
       "skippedTests" : [
         "EditorAztecTests",
-        "EditorGutenbergTests",
         "EditorTests\/testPlayground()",
         "LoginTests\/testEmailMagicLinkLogin()",
         "LoginTests\/testEmailPasswordLoginLogout()",
         "LoginTests\/testSelfHostedUsernamePasswordLoginLogout()",
         "LoginTests\/testUnsuccessfulLogin()",
         "LoginTests\/testWpcomUsernamePasswordLogin()",
-        "MainNavigationTests",
-        "ReaderTests",
         "SignupTests\/testEmailSignup()",
-        "StatsTests",
-        "SupportScreenTests",
```

Comment:
Enabled to force a failure on CI and validate the behavior

⚠️  I'll make sure to revert this commit once CI fails, to go back to green before we merge the PR.

Reply:
Reverted since, via [4b78c19](https://github.com/wordpress-mobile/WordPress-iOS/pull/17967/commits/4b78c196dfbe89b2a21462054119c1ad3f47a9cd)